lib/menu.ts

```diff
@@ -32,6 +37,24 @@ export interface IMenuOptions extends IPopupOptions {
  *      menuItem(...),
  *      menuItem(...),
  *    ]))
+ *
+ * The appearance of the menu and menuItem components can be changed by setting the following
+ * css variables in the parent project:
+ *
+ *    --weaseljs-font-size
+ *    --weaseljs-font-family
+ *    --weaseljs-background-color
```

Comment:
Actually I think all the ones that apply to the parent should be dropped, since they can be set directly. I.e. leave only the three that are used by menuItem.